test/js/ez-browse.js

```diff
@@ -0,0 +1,82 @@
+describe('ez-browse', function() {
+    let element;
+
+    beforeEach(function () {
+        element = fixture('BasicTestFixture');
+    });
+
+    function simulateClick(element) {
+        const click = new CustomEvent('click', {
+            bubbles: true,
+            cancelable: true,
+        });
+
+        element.dispatchEvent(click);
+        return click;
+    }
+
+    it('should be defined', function () {
+        assert.equal(
+            window.customElements.get('ez-browse'),
+            element.constructor
+        );
+    });
+
+    describe('properties', function () {
+        describe('`selectedLocationId`', function () {
+            it('should be set by the attribute', function () {
+                const value = 69;
+
+                element.setAttribute('selected-location-id', value);
+                assert.strictEqual(element.selectedLocationId, value);
+            });
+        });
+    });
+
+    describe('click', function () {
+        it('should dispatch `ez:contentDiscover`', function () {
+            element.addEventListener('ez:contentDiscover', (e) => {
+                assert.equal(e.detail.config.title, 'Browse', 'A title should be provided');
+                assert.equal(e.detail.config.startingLocationId, element.selectedLocationId, 'A startingLocationId should be provided');
+                assert.equal(e.detail.config.confirmLabel, 'View this content', 'A confirmLabel should be provided');
+                assert.isFunction(e.detail.listeners['ez:confirm'], 'An `ez:confirm` event listener should be provided');
+                assert.isTrue(e.bubbles, 'event should bubble');
+            });
+            simulateClick(element);
+        });
+
+        it('`ez:contentDiscover` should bubble', function () {
+            document.addEventListener('ez:contentDiscover', (e) => {
+                assert.isTrue(e.bubbles, 'event should bubble');
```

Comment:
no need to test that, listening on the document test the result of that. But you need to have a boolean to actually check that the event bubbled to the document.